server_structs/registry.go

```diff
@@ -44,8 +44,8 @@ type AdminMetadata struct {
 	UserID                string             `json:"user_id" post:"exclude"` // "sub" claim of user JWT who requested registration
 	Description           string             `json:"description"`
 	SiteName              string             `json:"site_name"`
-	Institution           string             `json:"institution" validate:"required"` // the unique identifier of the institution
-	SecurityContactUserID string             `json:"security_contact_user_id"`        // "sub" claim of user who is responsible for taking security concern
+	Institution           string             `json:"institution" validate:"required"`                                                                                // the unique identifier of the institution
```

Comment:
There's a strangely large amount of space before the comment here. Is that some hack to deal with linting or can we reduce it?